job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -183,6 +185,54 @@ BEGIN
          ELSE quote_ident(in_sort_field)
        END ||
         ' ' || CASE WHEN in_sort_ascending THEN 'ASC' ELSE 'DESC' END;
-    RETURN QUERY EXECUTE sql;
+
+    -- Create temporary table as a base to update the job progress
+    EXECUTE 'CREATE TEMPORARY TABLE get_job_temp ON COMMIT DROP AS ' || sql;
+
+    ALTER TABLE get_job_temp ADD COLUMN id SERIAL PRIMARY KEY;
+
+
+    FOR jobId IN SELECT * FROM get_job_temp
```

Comment:
I see two `FOR` statements but only one `END LOOP`.  Surely this can't be working?